### ISSUE TITLE
Confirm first paragraph in Data and Assessment

### ISSUE THREAD

**Issue by andrew-edwards** [cgrandin/hake-assessment#208]
Removed the following from the first paragraph in Exec 'Data and Assessment' section, because it doesn't seem to make sense:

"Total catch and empirical weight-at-age for \Sexpr{last.data.yr} are also added to the assessment model this year, but are fixed and not included in the model fitting procedure. "

Have re-written that paragraph also. See commit   279fcce
Highlighting as an issue in case I'm missing something and that sentence is needed.

**Comment by taylori** [comment]
I think that the deleted sentence was trying to draw a distinction between data which had a specific likelihood contribution that the model was fit to, and other model inputs that influence the dynamics but may or may not be considered "data". Those other inputs are the total catch at the top of the data file and the mean weight at each age defined in the wtatage.ss file.

I personally don't consider the distinction to be something that is important to make, especially when it leads to conclusion to I support removing that sentence.

I'm going to commit a potential revision to this paragraph which notes that the 1995 survey age comps were also added, and the 1998 point was slightly revised. If these changes make it seem too wordy for the Exec Summary, I'm happy to scale them back or let any of the rest of you attempt to wordsmith further (or leave that to @aaronmberger as our lead author).

**Comment by aaronmberger** [comment]
The paragraph that Andy added and then Ian added to looks good to me.  I'll wait for Chris to close this though as there is a merge conflict between Andy and Ian and my inexperience will screw something up no doubt if I try to remedy...

**Comment by taylori** [comment]
I think I've sorted out the merge conflict so I'm going to close the issue.
